packages/sdks/src/blocks/form/form.lite.tsx

```diff
@@ -1,333 +1,351 @@
-import { useMetadata } from '@builder.io/mitosis';
+import type { Signal } from '@builder.io/mitosis';
+import {
+  For,
+  Show,
+  useMetadata,
+  useRef,
+  useStore,
+  useTarget,
+} from '@builder.io/mitosis';
+import Block from '../../components/block/block.lite.jsx';
+import Blocks from '../../components/blocks/blocks.lite.jsx';
+import { isEditing } from '../../functions/is-editing.js';
+import { set } from '../../functions/set.js';
+import type { BuilderBlock } from '../../types/builder-block.js';
+import { filterAttrs } from '../helpers.js';
+/**
+ * This import is used by the Svelte SDK. Do not remove.
+ */
+
+import { builder } from '@builder.io/sdk';
```

Comment:
This SDK is part of gen1 and is not used by gen2 in any way.

Any usage needs to be replaced/migrated to the equivalent in gen2